### ISSUE TITLE
Getting error at the time of project compilation and not able to compile app

### ISSUE THREAD

**Issue by kamleshwebtech** [bluefireteam/photo_view#460]
Getting error at the time of project compilation and not able to compile app

```
../../flutter/.pub-cache/hosted/pub.dartlang.org/photo_view-0.11.1/lib/src/core/photo_view_gesture_detector.dart:106:29: Error: The argument type 'PointerEvent' can't be assigned to the parameter type 'PointerDownEvent'.
 - 'PointerEvent' is from 'package:flutter/src/gestures/events.dart' ('../../flutter/packages/flutter/lib/src/gestures/events.dart').
 - 'PointerDownEvent' is from 'package:flutter/src/gestures/events.dart' ('../../flutter/packages/flutter/lib/src/gestures/events.dart').
    super.addAllowedPointer(event);
```

Kindly suggest what is the issue and how can we fix it . Thanks a lot.

**Comment by tylyo** [comment]
same issue for me. do we have news?

**Comment by renancaraujo** [comment]
can we use 0.12.0?

**Comment by renancaraujo** [comment]
Seem like a dupe of https://github.com/bluefireteam/photo_view/issues/459 which was fixed already on 0.12.0
